### PR TITLE
Migration guide to 0.35 implementations syntax

### PR DESCRIPTION
**Pull request by evgenykochetkov**

### CODE REVIEW

**Review by nkrkv — APPROVED** [review]
:heart:

**Comment by gabbapeople** [comment]
👀

File: docs/guide/migrating-to-v035/README.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+  // ...
+}
+```
+
+After:
+```cpp
+node {
+    void evaluate(Context ctx) {
+        // much simpler syntax,
+        // available everywhere inside `node` struct
```
Comment (brusherru) [inline docs/guide/migrating-to-v035/README.md:167]: To make a clear comparison with "Before" and evident benefit, I suggest to add some example just inside `node`.

```
node {
  typeof_OUT a = {};
  void evaluate(Context ctx) {
    typeof_OUT b = {};
    // ...
  }
}
```

**Review by brusherru — APPROVED** [review]
Left one comment.
In the rest, LGTM.

File: docs/guide/migrating-to-v035/README.md (hunk 1 above)
Comment (evgenykochetkov) [reply docs/guide/migrating-to-v035/README.md]: Good idea! Done.

**Pull request merged**